fried-chicken.txt

```diff
@@ -0,0 +1,5 @@
+cure chicken with salt
+add egg, buttermilk, hot sauce, pickle juice
+add equal parts flour to kosher sale (cups to tsp)
+dunk chicken into flour, wet, flour
+fry
```

Comment:
Did you think about adding the temperature of the oil? What type of oil? Temperature of the chicken when it's done?